### DISCUSSION TITLE
使用 telnet 指令測 port 時，出現 zsh: command not found: telnet 的訊息

### DISCUSSION THREAD

**Discussion by KW0409** [Lidemy/forum#176] [其他問題] [answered]
今天在使用 telnet 指令的時候，偶然發現會出現 zsh: command not found: telnet 的訊息
(還是說只有我現在才發現....)
想說可能是這個指令不是預設會安裝的，就試著用 `npm install telnet` 來安裝
在 iTerm 下完指令也跑完 download 的進度條之後，發現還是不能使用 telnet 這個指令
後來看到這篇文章 [macOS 10.13 安装telnet方法](https://www.huaweicloud.com/articles/12642491.html) 才知道，要用 `brew install telnet` 的指令安裝
安裝完之後，就能正常使用了！
如果有錯誤的地方，還請各位多多指教，感謝～

**Comment by KW0409** [comment, marked as answer]
[macOS 10.13 安装telnet方法](https://www.huaweicloud.com/articles/12642491.html)

**Reply by aszx87410** [reply]
沒錯喔，要搞清楚 npm install xxx 是安裝 npm 套件的意思，而 brew install 才是一般安裝其他應用程式的指令

**Reply by KW0409** [reply]
！！原來如此
謝謝老師
